src/dmd/backend/type.c

```diff
@@ -209,10 +209,62 @@ unsigned type_alignsize(type *t)
     return sz;
 }
 
+/***********************************
+ * Compute special zero sized struct.
+ * Params:
+ *      t = type of parameter
+ * Returns:
+ *      true if it is
+ */
+bool type_zeroSize(type *t)
+{
+    if (config.exe & (EX_FREEBSD | EX_OSX))
+    {
+        /* Use clang convention for 0 size structs
```

Comment:
Isn't the auto-tester using GCC?

Reply:
AFAICT is clang used by FreeBSD:

https://github.com/braddr/d-tester/issues/70#issuecomment-368116189
https://github.com/braddr/d-tester/issues/70#issuecomment-368199964

Reply:
What about macOS?